lib/registry.js

```diff
@@ -28,10 +28,8 @@ var registry = (function() {
 		help = ['#', 'HELP', name, help].join(' ');
 		var type = ['#', 'TYPE', name, item.type].join(' ');
 
-		var values = (item.values || []).reduce(function(valAcc, val) {
-			var labels = Object.keys(val.labels || {}).map(function(key) {
-				return key + '="' + escapeLabelValue(val.labels[key]) + '"';
-			});
+		var values = (item.values || []).reduce((valAcc, val) => {
+			var labels = Object.keys(val.labels || {}).map((key) => key + '="' + escapeLabelValue(val.labels[key]) + '"');
```

Comment:
For me tis is harder to read now, split up the lines? Maybe prettier will mark that as an error?